build.gradle

```diff
@@ -160,8 +160,8 @@ task verifyVersions {
  * after the backport of the backcompat code is complete.
  */
 
-boolean bwc_tests_enabled = true
-final String bwc_tests_disabled_issue = "" /* place a PR link here when committing bwc changes */
+boolean bwc_tests_enabled = false
+final String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/39350" /* place a PR link here when committing bwc changes */
```

Comment:
What is the reason for disabling all BWC tests with this PR?

Reply:
BWC tests are currently being run against the 7.0.0 branch, which is using a previous lucene 8.0.0 snapshot.  There has been a change in the Codec format between these two snapshots, which means that an elasticsearch running with this change cannot read lucene indexes built with the previous snapshot; upgrade tests between the current 7.0.0 branch head and this PR fail because the indexes cannot be opened.

If there's a finer-grained BWC suspension I should be doing instead, please let me know!

Reply:
This also means that people using 7.0-alpha and 7.0-beta won't be able to do a rolling upgrade to 7.0-GA, because they're using unreleased lucene snapshots that do not contain this Codec change.

Reply:
This relates to #38919. The issue here being that that change is still under discussion. Let me see if we can accelerate it.

>This also means that people using 7.0-alpha and 7.0-beta won't be able to do a rolling upgrade to 7.0-GA, because they're using unreleased lucene snapshots that do not contain this Codec change.

That's okay, we do not support that to begin with. 😇

Reply:
I merged and back-ported #38919 . The discussion is still ongoing, but we should not hold anything up for it.